ds3-metadata/src/test/java/com/spectralogic/ds3client/metadata/MetadataAccessImpl_Test.java

```diff
@@ -142,7 +152,7 @@ public void onFailure(final FailureEvent failureEvent) {
                         }
                     },
                     "localhost")
-                    .getMetadataValue(filePath.toString());
+                    .getMetadataValue("forceAFailureByUsingANonExistentFileBecauseTheDockerImageRunsAsRoot");
```

Comment:
I'm not clear what this is trying to do, maybe the string name could be clearer? ;)